met-api/src/met_api/schemas/engagement.py

```diff
@@ -53,6 +54,14 @@ def get_submissions_meta_data(self, obj):
             }
         return {
             'total': len(obj.surveys[0].submissions),
+            'pending': len([submission for submission in obj.surveys[0].submissions
```

Comment:
Can we make this into a method by any chance?
a method to return the count if status is passed that way its more readable

`counts['pending'] = count_submissions_by_status(survey.submissions, CommentStatus.Pending)`
the method just does same thing in the above code

```
def count_submissions_by_status(submissions, status):
        return len([s for s in submissions if s.comment_status_id == status.value])
    
```

but since its already in return statement , i am not too sure how easy will be this.. but if we have a way to reduce the duplication , its great

Reply:
Done